packages/backend-output-storage/src/stack_metadata_output_storage_strategy.ts

```diff
@@ -2,14 +2,20 @@ import {
   BackendOutputEntry,
   BackendOutputStorageStrategy,
 } from '@aws-amplify/plugin-types';
-import { CfnOutput, Stack } from 'aws-cdk-lib';
+import { CfnOutput, Lazy, Stack } from 'aws-cdk-lib';
+
+// Aliased strings for readability
+type MetadataKey = string;
+type OutputKey = string;
 
 /**
  * Implementation of BackendOutputStorageStrategy that stores config data in stack metadata and outputs
  */
 export class StackMetadataBackendOutputStorageStrategy
   implements BackendOutputStorageStrategy<BackendOutputEntry>
 {
+  private lazyLists: Map<MetadataKey, Map<OutputKey, string[]>> = new Map();
```

Comment:
```suggestion
  private lazyListValueMap: Map<MetadataKey, Map<OutputKey, string[]>> = new Map();
```
Or whatever you think makes the most sense